contracts/hackatom/examples/schema.rs

```diff
@@ -24,4 +27,29 @@ fn main() {
 
     // state
     export_schema(&schema_for!(State), &out_dir);
+
+    // The new IDL
+    let path = out_dir.join("api.json");
```

Comment:
This should be `<crate_name>.json` (like how we name the wasm files in the rust-optimiser) to make it easier when copying into a common directory.